content/blog/get-started-with-prometheus-and-grafana-on-docker-with-hpe-storage-array-exporter.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Get started with Prometheus and Grafana on Docker with HPE Storage Array
+  Exporter
+date: 2021-12-07T16:00:00.000Z
+author: Michael Mattsson
+authorimage: /img/portrait-192.jpg
+tags:
+  - HPE Alletra
+  - HPE Nimble Storage
+  - Kubernetes
+  - Prometheus
+---
+With the recently released HPE Storage Array Exporter for Prometheus and HPE CSI Info Metrics Provider for Prometheus, it's a good time to familiarize ourselves with the cloud native technologies involved and get some first hand experience.
+
+Prometheus is a time-series database that also provides monitoring and alerting. It's a [CNCF graduated project](https://www.cncf.io/projects/). Grafana is a web-based visualization tool that uses time-series data to create beautiful graphs and elements to present data in views referred to as dashboards. Prometheus scrapes an HTTP endpoint of a target periodically to consume an assortment of metric types and and metadata. The target is usually referred to as an exporter and the data being scraped is the current state of one or many exporter data points.
+
+![Prometheus Overview](/img/prometheus-1.0.0-beta-revwip.png)
+
+In this tutorial we'll learn how to deploy Prometheus, Grafana and the HPE Storage Array Exporter for Prometheus using nothing but Docker. A supported storage backend, such as a HPE Alletra, Nimble Storage, Primera or 3PAR is needed to gather metrics for visualization. These prerequisites are assumed along with basic docker and container knowledge. While this is a highly opinionated tutorial, the different components may be run standalone on Windows or Linux and may also be deployed directly on Kubernetes with Helm.
+
+# Get started
+
+The tutorial assumes docker is installed and functional. Credentials are also needed for a backend system. HPE Nimble Storage is being used in the examples and metrics may be named differently when using something else. Still, principles remain the same.
+
+To allow the containers to communicate with each other by name, a separate bridge network is created.
+
+```markdown
+docker network create prometheus
+```
+
+Note that all containers created below do not utilize any persistent storage at all. Any dashboards that have been created or metrics that have been gathered will not be accessible after a container restart.
+
+**Note:** This tutorial is purely for educational purposes and not intended for production deployments.
```

Comment:
This is a good note. I'd recommend putting it closer to the beginning of the post, perhaps at the end of the paragraph just before "Get started" where it says "...on Kubernetes with Helm."